### PR TITLE
frontend/templates: remove old information

### PR DESCRIPTION
**Pull request by LemmeCry**
Removes old information about additional payment methods and ad spaces, edits the guild join card to show the link to the docs instead of saying they're incomplete, fixes general wording.

### CODE REVIEW

File: frontend/templates/cp_selectserver.html [hunk 1]
```diff
@@ -11,8 +11,8 @@
                 <h2 class="card-title">Bot added to <b>{{.JoinedGuild.Name}}!</b></h2>
             </header>
             <div class="card-body">
-                <p>You manage this bot through a web control panel, if you need any help or want to contact the owner you can join the support server (There is a link above)</p>
-                <p>I'm working on adding proper documentation and guides but at this moment the documentation is very lacking, and it may seem a bit complicated. As as I said earlier, join the support server if you have any questions, although please do try a little on your own first, as I and the support team's time is limited.</p>
+                <p>You manage this bot through a web control panel, if you need any help you can join the support server (There is a link above)</p>
+                <p>The documentation for the bot can be found at <a class="link" href="https://docs.yagpdb.xyz/">docs.yagpdb.xyz</a>. As stated earlier, join the support server if you have any questions, although please do try a little on your own first, as the support team's time is limited.</p>
```
Comment (l-zeuch) [inline frontend/templates/cp_selectserver.html:15]:
```suggestion
                <p>The documentation for the bot can be found at <a class="link" href="https://docs.yagpdb.xyz/">docs.yagpdb.xyz</a>. Join the support server if you have any questions, although please do try a little on your own first, as the support team's time is limited.</p>
```

**Review by l-zeuch — CHANGES_REQUESTED** [review]

**Review by 3kh0 — APPROVED** [review]
Looks good

File: frontend/templates/cp_selectserver.html [hunk 2]
```diff
@@ -80,9 +80,8 @@ <h2 class="card-title">Thanks to my patrons!</h2>
                 <h2 class="card-title">Check out...</h2>
             </header>
             <div class="card-body">
-                <p>YAGPDB is open source! The project is hosted on github here: <a href="https://github.com/botlabs-gg/yagpdb">jonas747/yagpdb</a>.</p>
-                <p><a href="https://docs.yagpdb.xyz/donating" target="_blank">Donate using patreon, bitcoin and other cryptos!</a> Donating $3 or more will grant you premium slots you can assign to servers!</p>
-                <p>Looking to buy the ad space? <a href="https://docs.yagpdb.xyz/donating#advertisement" target="_blank">Click here for more info</a>.</p>
+                <p>YAGPDB is open source! The project is hosted on github here: <a href="https://github.com/botlabs-gg/yagpdb">botlabs-gg/yagpdb</a>.</p>
```
Comment (jo3-l) [inline frontend/templates/cp_selectserver.html:83]:
```suggestion
                <p>YAGPDB is open source! The project is hosted on GitHub here: <a href="https://github.com/botlabs-gg/yagpdb">botlabs-gg/yagpdb</a>.</p>
```

**Review by jo3-l — COMMENTED** [review]
nit, but might as well fix the capitalization of github

**Pull request merged**